drafts/25-xxx-macro-expansion.txt

```diff
@@ -182,26 +229,83 @@ EXAMPLE age04:
 within function-like macro invocations in free source form.
 
-lb01. An invocation of a function-like macro can be broken across
-      multiple lines. An invocation that begins with an open parenthesis
-      '(' continues until the matching closing parenthesis ')' is found
-      (outside of a comment), which may occur on a subsequent source
-      fragment line.
+lb01. [Straw poll 2, option 1] An invocation of a function-like macro
+      can be continued across multiple lines. An invocation that begins
+      with an open parenthesis '(' continues until the matching closing
+      parenthesis ')' is found (outside of a comment).
+
+lb01. [Straw poll 2, option 2] An invocation of a function-like macro
+      can be broken across multiple lines. An invocation that begins
+      with an open parenthesis '(' continues until the matching closing
+      parenthesis ')' is found (outside of a comment), which may occur
+      on a subsequent source fragment line.
 
 lb03. Comment lines within the argument list of an invocation of a
       function-like macro are discarded (section 2.1.4).
 
 lb05. A newline character appearing within the argument list is treated
       as a blank character, except where rule lb07 applies.
 
-lb07. Fortran-style continuation markers are permitted. An ampersand '&'
-      which is the last non-whitespace character (after the removal of
-      comments) preceding a newline character is removed, and all
-      subsequent characters up to and including the newline are removed.
-      If the first non-whitespace character on a continuation line is an
-      ampersand '&', all characters up to and including the leading
-      ampersand '&' are removed.
+lb07. [Straw poll 2, option 1] Fortran-style continuation markers are
+      required. An ampersand '&' which is the last non-whitespace
+      character (after the removal of comments) preceding a newline
+      character is removed, and all subsequent characters up to and
+      including the newline are removed. If the first non-whitespace
+      character on a continuation line is an ampersand '&', all
+      characters up to and including the leading ampersand '&' are
+      removed.
+
+lb07. [Straw poll 2, option 2] Fortran-style continuation markers are
+      permitted. An ampersand '&' which is the last non-whitespace
+      character (after the removal of comments) preceding a newline
+      character is removed, and all subsequent characters up to and
+      including the newline are removed. If the first non-whitespace
+      character on a continuation line is an ampersand '&', all
+      characters up to and including the leading ampersand '&' are
+      removed.
+
+[Straw poll 2, option 1]
+EXAMPLE lbe01:
+
+      If MAC is a function-like macro which accepts three arguments,
+      then the following lines:
+
+            MAC(a,
+                b,
+                c)
+            MAC(q,r
+            s,
+            t)
+
+      are erroneous.
```

Comment:
The standard basically never says anything is "erroneous". 
In particular, we don't want to _require_ an error for this case, because (despite specifying option 1) some processors may still choose to deploy this as an extension.

Strawman replacement:
```suggestion
      have a processor-dependent interpretation.
```
but there is probably a better way to state what I think we want, which is that such lines have no specified meaning but processors are not required to reject the program or generate an error diagnostic.